### ISSUE TITLE
How to add an image description for toot in Whalebird 5+ ?

### ISSUE THREAD

**Issue by TearOfTheStar** [h3poteto/whalebird-desktop#4199]
Can't find this option anywhere.

**Comment by h3poteto** [comment]
Yes.

**Comment by tigexplorer** [comment]
For accessibility reasons, unfortunately the lack of this option makes Whalebird for me currently unusable when i'ts come to posting images.

**Comment by h3poteto** [comment]
Please check 6.0.0.
https://github.com/h3poteto/whalebird-desktop/releases/tag/v6.0.0
